Makefile

```
@@ -42,6 +42,9 @@ gxdr/xdr_generated.go: $(XDRS)
 xdr/%.x:
 	curl -Lsf -o $@ https://raw.githubusercontent.com/stellar/stellar-xdr-next/$(XDRNEXT_COMMIT)/$(@F)
```

Note that the repo is renamed.
```suggestion
	curl -Lsf -o $@ https://raw.githubusercontent.com/stellar/stellar-xdr/$(XDRNEXT_COMMIT)/$(@F)
```

---

yes, I'm well aware of that. However, @paulbellamy PR already addresses exactly that.. so I don't want it in my PR.